### ISSUE TITLE
Desktop / Browser: Accessibility - Custom field reordering not keyboard accessible

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#2658]
### Steps To Reproduce

1. Go to a cipher/item with more than one custom field
2. Edit the item
3. Note the ability to reorder the custom fields (the three-line "grabber" control)

### Expected Result

It should be possible to reorder fields using the keyboard.

### Actual Result

Currently, custom field reordering only works with a mouse/touchscreen. The three-line reordering control is not keyboard focusable, and there are no alternative controls to move an item up/down.

### Screenshots or Videos

The issue is present both on the browser extension

![a series of custom fields, with one field currently being dragged with the mouse to reorder it](https://user-images.githubusercontent.com/895831/167274554-ff55d009-08a6-4370-a782-a7978b912068.png)

and on the desktop application

![a series of custom fields, with one field currently being dragged with the mouse to reorder it](https://user-images.githubusercontent.com/895831/167274568-a0df98fd-888c-494b-becc-d87f3934e18c.png)

### Additional Context

One potential approach is to make the reordering drag handle three-lines control keyboard focusable, and once it does receive focus, display two additional focusable arrow items (arrow up, arrow down) that a keyboard user can focus and then activate to move the item up or down one slot.

I would recommend NOT trying to implement this purely as a keyboard event handler checking for up/down cursor keys, as these will be ignored/overridden for users that also navigate via keyboard and a screen reader, as the cursor keys will be processed directly by - Windows-based - screen readers to move focus (while on macOS they would still work, since VoiceOver doesn't use pure cursor keys, but needs the VO key to be pressed as well)

Incidentally, 1Password shows two arrows (up and down) when editing an item that has multiple websites (rather than using drag'n'drop). But ironically, these controls in 1Password are also not keyboard focusable/operable. So the approach (showing these controls) is good, but needs to be taken further to actually still then make them work for keyboard users.

![screenshot from 1Password, showing two website fields - the one in focus is preceded by up and down arrow controls](https://user-images.githubusercontent.com/895831/167274747-8f85b991-22db-4b24-abad-ee06933073ec.png)

### Operating System

Windows

### Operating System Version

11

### Web Browser

Chrome

### Browser Version

101.0.4951.54 (Official Build) (64-bit)

### Build Version

1.58.0 (extension) / Version 1.33.0 Shell 16.1.0 Renderer 96.0.4664.174 Node 16.9.1 Architecture x64 (desktop)

**Comment by Krychaz** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development related issues. The Bitwarden Community Forums has a Feature Requests section for submitting, voting for, and discussing requests like this one: https://community.bitwarden.com/c/feature-requests/ 

Please sign up on our forums and search to see if this request already exists (https://community.bitwarden.com/signup). If so, you can vote for it and contribute to any discussions about it. If not, you can re-create the request there so that it can be properly tracked.

This issue will now be closed. Note that the “Closed as not planned” status is for internal tracking purposes only, please do not hesitate to send us your suggestions to the forum.

Thanks!

**Comment by patrickhlauke** [comment]
> We use GitHub issues as a place to track bugs and other development related issues.

not being accessible *is* a bug...but hey, you do you

**Comment by djsmith85** [comment]
Re-opening...

@Krychaz Any kind of reports with accessibility issues, should be kept open. 

Thanks for tagging me @patrickhlauke

**Comment by ariane-bitwarden** [comment]
With the refreshed desktop UI released in July, custom fields can now be reorganized using keyboard navigation with the up and down arrow keys. This appears to resolve the accessibility issue described in this report. If there remains an accessibility problem here, please do re-open this issue. Thanks!

**Comment by patrickhlauke** [comment]
@ariane-bitwarden the issue is still present. while yes, there is keyboard navigation on those reorder buttons ... they can't actually be operated when a (windows) screen reader is running, as cursor up/cursor down is used in the screen reader to move the reading cursor, and the key presses never make it to the actual JavaScript

Video of Bitwarden extension without, and then with, NVDA screen reader running - once NVDA runs, cursor up/down simply starts reading the previous/next controls/fields

https://github.com/user-attachments/assets/6887c210-7c97-43c2-92cd-be78780a049f

Same in dekstop (it's actually worse for different reasons, as the buttons are also unlabelled there)

Video of Bitwarden desktop with JAWS running - cursor up/down just reads the previous/next control

https://github.com/user-attachments/assets/bed74b62-3567-4787-b2ed-26a202cbc03f

Please either reopen this issue, or let me know if I should open a new fresh issue

/cc @djsmith85

**Comment by jengstrom-bw** [comment]
Did some investigating into this, and it appears to be possible for a windows user using NVDA or JAWS to toggle to a focus mode (`NVDA+space` or `JAWS+z`) that allows the arrow keys to bypass the screen reader and get to the application. #20099  adds an `aria-description` that will prompt a screen reader user to enter this focus mode in order to reorder these items strictly using a keyboard.

**Comment by patrickhlauke** [comment]
Nice work
